src/Workspaces/Core/Desktop/SymbolSearch/SymbolSearchUpdateEngine.cs

```diff
@@ -108,7 +108,25 @@ public SymbolSearchUpdateEngine(ISymbolSearchLogService logService)
             return Task.FromResult(result.ToImmutableAndFree());
         }
 
-        public Task<ImmutableArray<PackageWithAssemblyResult>> FindPackagesWithAssemblyAsync(
+        public async Task<ImmutableArray<PackageWithAssemblyResult>> FindPackagesWithAssemblyAsync(
+            string source, string assemblyName)
+        {
+            var result = await FindPackagesWithAssemblyWorkerAsync(source, assemblyName).ConfigureAwait(false);
+
+#if DEBUG
+            // For testing purposes, we hardcode this in if we're in DEBUG mode.
```

Comment:
What about when I'm in debug but not running tests? Should there be a better test hook here?